commons/src/config/mod.rs

```diff
@@ -0,0 +1,75 @@
+use std::{path::PathBuf, collections::{HashMap, HashSet}};
+
+
+
+pub struct Config {
+  pub preamble: Preamble,
+  pub merging: MergingConfig,
+  pub input: PathBuf,
+}
+
+pub struct MergingConfig {
+  pub ignore_preamble: bool,
+}
+
+pub struct Preamble {
+  pub output: Output,
+  pub metadata: Metadata,
+  pub cite: Citedata,
+  pub render: RenderConfig,
+  pub i18n: I18n,
+}
+
+pub struct I18n {
+  pub lang: Option<String>,
+  pub langs: HashSet<String>,
+}
+
+
+pub struct RenderConfig {
+  /// K = element/attribute name, V = `true` to ignore
+  pub ignore: HashMap<String, bool>, 
+  pub parameter: HashMap<String, String>,
+  pub keep_comments: bool,
+  pub non_strict: bool,
```

Comment:
I would avoid negation here: 

```suggestion
  pub strict: bool,
```

Reply:
non-strict is intentional here, because „strict“ should be the default, and setting the option marks non-strict as „true“.

Reply:
Any chance that we come up with a better name? What effect does this have on rendering?

Reply:
The effect is similar to `ignore`, but an implementation (such as `unimarkup-rs`) may choose types/elements that are **not safe**. These types/elements are then per default ignored on rendering, and are only allowed if `non-strict` is set to `true`.

e.g.: we may define `script` HTML tags as **not safe** to ensure no JavaScript is in the rendered HTML output.

Reply:
What if we call this: `render_unsafe` or `allow_unsafe`? 

`non_strict` is a negation, so someone sets the negation to true. I find that a little confusing. It works as well, we can leave it, but I think using some sort of positive statement works better.

Reply:
`allow_unsafe` seems fitting. In the specification I even start with `Allows unsafe features...` for the description of this option.